_posts/blog/2018-11-02-el-topo-blindado.md

```diff
@@ -0,0 +1,59 @@
+
+*Organizaciones Político-Militares* (OPM) is the broad name given to those groups whose political vision for a different and better social organization was wedded to their commitment to armed struggle as the vehicle through which to bring about these changes. In Argentina, the two most studied of these groups are the *PRT-ERP* and the *Montoneros*. They were, however, only two of the groups whose activities contributed to the broader phenomenon of OPM's. *El Topo Blindado: Centro de documentación de las organizaciones politico-militares argentinas* is the latest manifestation of an investment made by a collective of Argentine researchers (*El Topo Blindado*) - committed to the study of the role(s) played by OPM's in the recent and violent history of Argentina - to promote the deeper and more dynamic exploration of this broader phenomenon beyond the privileging of these two organizations and beyond the reduction of the activities of these kinds of groups to individual actions, pathologies, etc. 
+
+The first manifestation of this commitment would see the collective conduct workshops and reading groups in order to create spaces for a debate on the role these groups committed to armed struggle: a debate that would take as its starting point the material *produced by* these OPM's. These efforts would find a natural expression in the production - between 2004 and 2008 - of *Lucha Armada en la Argentina*: a magazine publication that sought to think through the role of these groups in Argentine contemporary history. By 2010 it became clear to the collective that in order to achieve their goal of promoting the deeper study of these groups and their history, an archive of these sources would need to be publicly established and made *freely* available to any person who might wish to explore and study this history. This insight meant taking advantage of the technology made available by the contemporary world: the internet. 
+
+The original exploration of this contemporary medium would take the form of a blog - of the same name - that would begin the process of making these primary sources available to the general public. The blog itself functioned much like the magazine: as both a space for the continuation of the debate and as an invitation to the general public (or those with an interest) to participate. What the digital format provided that the magazine platform did not, was the ability to store and make available digital scans of the primary sources. As the archive grew, however, it became clear to the collective that the blog platform was insufficient and somewhat cumbersome, and so in 2011 the blog site transformed into the site that is being reviewed here. 
```

Comment:
These two paragraphs, describing the history of the project, can be condensed and switched into past tense (rather than using "would"). Try something like this: 

El Topo Blindado is based on over a decade of work by the Argentine collective, which included workshops, a print magazine, and a blog. [brief description.']